source/managing_account/02_managing_api_keys/managing_api_keys.rst

```diff
@@ -11,9 +11,9 @@ These API keys allow you to use the following APIs: ODS Search API v1 and v2, OD
    :class: important
 
    API keys inherit all permissions granted to the user for which it has been created.
-   That's why users should be cautious with how they use API keys.
+   That's why you should be cautious with how they use API keys.
    
-   If you observe unauthorized usage of an API key, if an API key gets exposed or is unused, it is recommended to revoke it and create a new one.
+   If you observe unauthorized usage of an API key or if an API key gets exposed, it is recommended to contact Opendatasoft support to get help identifying and revoking leaked API keys.
```

Comment:
This phrase should be in the `01_managing_users_and_groups/users.rst` file and the previous phrase should stay here.
A user can manage and revoke their own key, however, administrators cannot manage keys on behalf of someone. Therefore if an API key is leaked:
* and the owner of the API key noticed it, they can revoke it themselves (=> for this file)
* and the admin of the concerned domain notices it they can only ask ODS support for help because they don't know who the key belongs to and they have no means of revoking it on their behalf (=> for the `01_managing_users_and_groups/users.rst` file)

Reply:
Fixed in 42df573a21bf46603d13f707beb12e11b6fbc800